common/mq/publisher.js

```diff
@@ -12,8 +13,14 @@ const log = require('@idearium/log')();
 const publish = (type, data) => {
 
     return manager.publish(type, data)
-        .then(() => log.debug({ data, type }, `Published message of type: ${type}`))
-        .catch(err => log.error({ err }, `Could not publish message of type: ${type}`));
+        .then(() => multiLog(
+            {
+                trace: { data },
```

Comment:
@allanchau, this is good, but I think it should be at debug.
```suggestion
                debug: { data },
```